.yeoman/templates/containers/src/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const <%= componentName %> = () => <h1>Hello World</h1>;
```

Comment:
If this variable needs to be the same across all the different templates, perhaps it should be renamed to `packageName` rather than `componentName`?